planetaryimage/decoders.py

```diff
@@ -3,9 +3,10 @@
 
 
 class BandSequentialDecoder(object):
-    def __init__(self, dtype, shape, compression=None):
+    def __init__(self, dtype, shape, sample_bytes, compression=None):
```

Comment:
Here you change the calling signature of `BandSequentialDecoder` which is also used by `cubefile`.

The only test for cubefiles is of the image `data/pattern.cub` which, according to the following label, will use the `TileDecoder`, so I suspect that when cubefile encounters its first band sequential product it will explode.

```
Object = IsisCube
  Object = Core
    StartByte   = 65537
    Format      = Tile
    TileSamples = 128
    TileLines   = 128

    Group = Dimensions
      Samples = 90
      Lines   = 90
      Bands   = 1
    End_Group

    Group = Pixels
      Type       = Real
      ByteOrder  = Lsb
      Base       = 0.0
      Multiplier = 1.0
    End_Group
  End_Object
End_Object

Object = Label
  Bytes = 65536
End_Object
End
```